website/tutorial/part_2/part_2a/08_todo_service.md

```diff
@@ -9,6 +9,12 @@
 
 # Building the Todo service
 
+> Note: This is the second service of the tutorial. 
```

Comment:
Do we have to say that here? I think we can assume that people follow a tutorial correctly. This could be said on every page.

Reply:
The reason I stated this here is because this page relies on concrete changes from previous pages. This page instructs you to select template number 2. If template number 1 has not been added then it makes this page function incorrectly. Basically, I added it to mitigate confusion in case anyone finds their way to this page without following the appropriate steps.

If you still feel it's unnecessary considering this then I have no problem removing it.

Reply:
I think we can assume that people aren't idiots, and can follow a tutorial correctly as long as all changes we want them to do are easily visible and clearly spelled out. If there are particular concerns that people will get tripped up here, maybe we can provide a working copy of the code base for this stage somewhere in a folder? That way, if our readers don't get it working, they can look at our code and see what is different.